test/screenshot/top-app-bar/standard.js

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
 import TopAppBar from '../../../packages/top-app-bar';
-import '../../../packages/top-app-bar/index.scss';
-import '../../../packages/material-icon/index.scss';
-import './index.scss';
-
 import MaterialIcon from '../../../packages/material-icon';
+import classnames from 'classnames/bind';
+
+import styles from './test1.scss';
+const cx = classnames.bind(styles);
```

Comment:
if styles are obfuscated via css modules, them in this `bind` method to get a mapping of it. Then we can use it below:

```
cx('demo') // gets the obfuscated class name
```